data/i18n/en/en.toml

```diff
@@ -372,3 +372,75 @@ other = "Warning:"
 
 [whatsnext_heading]
 other = "What's next"
+
+[canonical_tag_architecture_name]
```

Comment:
```suggestion
[layouts_docs_glossary_architecture_name]
```
This will make it more obvious what context these are used in, and matches an existing glossary entry.

Reply:
Does it also apply to the descriptions?

Reply:
I preemptively also applied this to the descriptions, but can revert the change if needed.